### PR TITLE
Issue #SB-22937 feat: Onboard user list API

### PR DESCRIPTION
**Pull request by ahghatol**

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -5946,6 +5946,24 @@ kong_apis:
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: userList
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5949]: Is this API documented @anmol2302

Comment (ahghatol) [reply ansible/roles/kong-api/defaults/main.yml]: @SMYALTAMASH PR sent to dock repo: https://github.com/project-sunbird/sunbird.org-docs/pull/736

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -5946,6 +5946,24 @@ kong_apis:
...
+    request_path: "{{ program_service_prefix }}/v1/contributor/search"
+    upstream_url: "{{ program_service_url }}/program/v1/contributor/search"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'userTempAdmin'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5959]: Invalid ACL

Comment (SMYALTAMASH) [reply ansible/roles/kong-api/defaults/main.yml:5959]: as per the discussion from @pallakartheekreddy and @ahghatol admin will search for multiple users in a single API call and display the necessary details, 

Approving it to have userTempAdmin ACL.

**Review by SMYALTAMASH — APPROVED** [review]

**Pull request merged**